oprf/group/suite.go

```diff
@@ -0,0 +1,201 @@
+package group
+
+import (
+	"crypto/elliptic"
+	"crypto/rand"
+	"errors"
+	"io"
+	"math/big"
+
+	"github.com/armfazh/h2c-go-ref"
+)
+
+// Ciphersuite corresponds to the OPRF ciphersuite that is chosen. The
+// Ciphersuite object determines the prime-order group (pog) that is used for
+// performing the OPRF operations, along with the different hash function
+// definitions.
+// Should be created using NewSuite, using an string.
+type Ciphersuite struct {
```

Comment:
An OPRF ciphersuite also requires a specific hash function -- should we add that here?

Reply:
Yes.. I thought so as well. But it seems like they all use `SHA-512`?

Reply:
For now :) We will likely change this in a version soon.

Reply:
Ah, gotta it. I'll put it there ;)